pysat/utils/io.py

```diff
@@ -670,6 +709,8 @@ def load_netcdf_xarray(fnames, strict_meta=False, file_format='NETCDF4',
         data = xr.open_mfdataset(fnames, decode_timedelta=decode_timedelta,
                                  combine='by_coords')
 
+    # TODO(#947) Add conversion for timestamps, including origin and unit
```

Comment:
```suggestion
    # TODO(#947) Add conversion for timestamps that may have been treated incorrectly,
    # including origin and unit.  At the moment this is only done for Pandas data.
```

Tried to clarify this comment, since it confused me and I had to go and read the issue to make any sense of it.